### ISSUE TITLE
Overgrowth issues with minecolonies

### ISSUE THREAD

**Issue by Estyria** [ldtteam/minecolonies#8901]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Verison?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### Minecraft Version

1.19

### MineColonies Version

1.19.2-1.0.1198

### Structurize Version

1.19.2-1.0.476

### Related Mods and their Versions

Holiday server

### Current Behavior

Issues with Overgrowth at the moment:

-doors pop off (not sure if this got fixed yet honestly)
-pressure plates pop off
-herders are turning grass into paths in the pens at an alarming rate still
-plantation has no way to fix sand turning into sandstone, and sugar cane ain't happy

Additional items that I think need adjusted:

-spider webs should not spawn within five blocks of another spider web
-grass is still growing crazy fast
-Overall I'd probably turn down the conversion rates a good chunk more to exit the realm of annoying and enter the realm of interesting feature

### Expected Behavior

Builds should not be broken by this in any way.

### Reproduction Steps

Just play a while, it'll all start happening.

### Logs

https://pastebin.com/R3GF5Frc

### Anything else?

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by kezmodius** [comment]
Agreed. The grass growth rate and web spawns are still far too high. The sheep pen is full of paths from the herder.
It is making Minecolonies less enjoyable to play as I have to constantly focus on upkeep or everything looks terrible.
The odd little patch of grass that grows is fun. Having every block of grass be needing to be mown multiple times a play session is not fun.
